src/openms_gui/include/OpenMS/VISUAL/LayerData.h

```diff
@@ -82,7 +82,7 @@ namespace OpenMS
 
   @ingroup SpectrumWidgets
   */
-  class LayerData
+  class OPENMS_GUI_DLLAPI LayerData
```

Comment:
maybe a good place for polymorphism in the future? to have different layer types  for different data?

Reply:
Yes! People have been adding data members in the past without thinking about this... so currently, its a mess.

Reply:
Ok I'll add an issue for that